components/Search/Hit.tsx

```diff
@@ -50,11 +51,23 @@ const TagsContainer = styled.div`
   gap: ${tokens.spacings.comfortable.small};
 `
 
+const TopContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  gap: ${tokens.spacings.comfortable.large};
+`
+
 const StyledChip = styled(Chip)`
   background-color: rgb(255, 218, 168);
   color: var(--charcoal);
 `
 
+const StyledChipNew = styled(Chip)`
+  background-color: var(--white);
+  color: var(--charcoal);
+  border: 2px solid rgb(255, 218, 168);
```

Comment:
Please don't add colours like this since all other colours are now as CSS variables

Reply:
This color did not exist in CSS variable. Have added it now and changed in Styled chip as well

Reply:
Done